src/main/java/org/opentripplanner/street/model/edge/StreetEdge.java

```diff
@@ -984,13 +1027,19 @@ private StateEditor doTraverse(State s0, TraverseMode traverseMode, boolean walk
       }
     }
 
-    var s1 = createEditor(s0, this, traverseMode, walkingBike);
+    var editor = createEditor(currentState, this, traverseMode, walkingBike);
```

Comment:
We should add notes somewhere what `editor` and `currentState` mean, s<sub>0</sub> and s<sub>1</sub> are terminology widely used by the A* algorithm. Also isn't s<sub>0</sub> the state at the from vertex, not really the current state?

Reply:
I've changed it back. I thought they were automatically generated names.

Reply:
We might still want to document the meaning of the variables somewhere